src/components/Banners/SiteBanner.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import IconButton from "@leafygreen-ui/icon-button";
+import Icon from "@leafygreen-ui/icon";
+import Cookies from "js-cookie";
+import get from "lodash.get";
+import { GET_SITE_BANNER } from "gql/queries";
+import { SiteBannerQuery } from "gql/generated/types";
+
+const { yellow, blue, green, red } = uiColors;
+export const SiteBanner = () => {
+  const { data, loading } = useQuery<SiteBannerQuery>(GET_SITE_BANNER);
+  const { text, theme } = get(data, "siteBanner", {});
+  const [showBanner, setShowBanner] = useState(false);
+  useEffect(() => {
+    if (text !== "" && Cookies.get(text) === undefined) {
+      setShowBanner(true);
+    }
+  }, [text]);
+  if (loading) return null;
+
+  const hideBanner = () => {
```

Comment:
The behavior in the old UI is to redisplay the banner as soon as a new banner with new text is set https://github.com/evergreen-ci/evergreen/blob/880b3fd3058567b85907ed0e0d45461f80c5eab9/public/static/js/menu.js#L36. Is that behavior supported here?

Reply:
Yup it is I took that into account. We set a cookie with the current banner text when they hide the banner if the banner text changes then the cookie wont match and we show the banner.

Reply:
Thanks. I'll remove myself as a blocker from the review.